deptofdefense/move.mil
# Corrections to transportation offices and weight scales

## Checklist

I have…


## Summary of Changes

This pull request…

- Removes several duplicate (and even triplicate) transportation offices and weight scales.
- Corrects the hours of operation for several offices.
- Corrects the address of the Malmstrom AFB office.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. git checkout `pppo_updates`,
1. set up development dependencies according to [CONTRIBUTING.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTING.md),
1. run `bin/rails server`, and
1. load up [http://localhost:3000](http://localhost:3000) in the Web browser of your choice.
